plugins/CoreHome/javascripts/dataTable.js

```diff
@@ -567,6 +566,11 @@ $.extend(DataTable.prototype, UIControl.prototype, {
 
             if (labelColumnWidth) {
                 $('td.label', domElem).each(function() {
+                    var tableClass =  $(this).closest('table').attr('class');
+                    // check if the table is sub table. remove the width by 20px to make sure the table size wont change.
+                    if (tableClass.includes('subDataTable')) {
+                        labelColumnWidth = labelColumnWidth - 20;
+                    }
```

Comment:
That seems to cause that the label column of subtables is smaller than the one of the parent table, which looks a bit ugly. Guess then it's better if the table becomes bigger when opening the subtable.